news/2017-01-04-tip4-testing.asciidoc

```diff
@@ -54,13 +74,19 @@ For example to use Arquillian Embedded Tomcat container you need to define follo
                 </dependency>
 ----
 
-== Java SE environment
+== Java SE environment & Arquillian Weld Embedded container
+In SE environment we can shake off all the complexity of other EE tehcnologies (EJB. JTA, JPA, ...) but we also have to realize that we are down to one "flat" JAR file.
+For such a situation, Arquillian Weld Embedded container is a perfect fit, so let's take a closer look
```

Comment:
Well I have slightly different idea for this topic. I will try to introduce my proposal.